tools/linter_lib/custom_check.py

```diff
@@ -140,7 +140,7 @@ def build_custom_checkers(by_lang):
          'description': 'The keyword "function" should be followed by a space'},
         {'pattern': '.*blueslip.warning\(.*',
          'description': 'The module blueslip has no function warning, try using blueslip.warn'},
-        {'pattern': '[)]{$',
+        {'pattern': '\){$',
```

Comment:
I guess the reason we've preferred using the `[)]` syntax historically is that while it has more characters, it feels more readable.